perfkitbenchmarker/data/cassandra/create-ycsb-table.cql.j2

```diff
@@ -12,10 +12,17 @@
  # See the License for the specific language governing permissions and
  # limitations under the License. #}
 {# Create a table for YCSB benchmark. #}
-create keyspace {{ keyspace }}
-  with placement_strategy = 'SimpleStrategy'
-  and strategy_options = {replication_factor: {{ replication_factor }} };
-
-use {{ keyspace }};
-
-create column family {{ column_family }};
+create keyspace ycsb WITH REPLICATION = {'class' : 'SimpleStrategy', 'replication_factor': {{replication_factor}} };
+USE ycsb;
+create table usertable (
```

Comment:
What do you mean by that? I should add a param to execution like --ycsb_field_count that will generate fields in table schema (with 10 as default)?